### DISCUSSION TITLE
Which Tokens to Allow Donations for MVP?

### DISCUSSION THREAD

**Discussion by apbendi** [dcgtc/dgrants#72] [Architecture & Design] [answered]
Dependent on contract implementation per #71. Other considerations:

* Security
* Liquidity
* UX / UI design

**Comment by apbendi** [comment, marked as answer]
Via @mds1, as of August 2020, DAI, ETH, USDC, and PAN accounted for 99.24% of donations on cGrants

Currently, PAN does not have liquidity on Uniswap v3, but we're working with them to get a pool established.

Additionally, cGrants has since added UNI, for which there should be plenty of liquidity.

Finally, we want to support donating directly in GTC, of course.

Therefore, I propose the following list of supported tokens on the frontend for the MVP:

* ETH
* DAI
* USDC
* GTC
* UNI

With PAN to be added iff their community can get sufficient liquidity on Uniswap v3.

**Comment by Tamara6969** [comment]
💪💪💪
